Tools/Gulp/config.json

```diff
@@ -4,7 +4,7 @@
     "minFilename": "babylon.js",
     "minNoWorkerFilename": "babylon.noworker.js",
     "declarationFilename": "babylon.d.ts",
-    "outputDirectory": "../../dist/",
+    "outputDirectory": "../../../Editor/dist/",
```

Comment:
This wasn't suppose to be posted to fork.... Still learning Git and i made a mistake here. This is my desitination